File: dodam-system-domain/dodam-domain-rds/src/main/java/b1nd/dodam/domain/rds/club/enumeration/ClubPriority.java
package b1nd.dodam.domain.rds.club.enumeration;

import java.util.List;

public enum ClubPriority {
    CREATIVE_ACTIVITY_CLUB_1,
    CREATIVE_ACTIVITY_CLUB_2,
    CREATIVE_ACTIVITY_CLUB_3;

    public static List<ClubPriority> getClubPriorities(){
        return List.of(CREATIVE_ACTIVITY_CLUB_1,  CREATIVE_ACTIVITY_CLUB_2, CREATIVE_ACTIVITY_CLUB_3);
    }
}

File: services/service-club/src/main/kotlin/com/b1nd/dodamdodam/club/domain/club/enumeration/ClubPriority.kt
package com.b1nd.dodamdodam.club.domain.club.enumeration

enum class ClubPriority {
    CREATIVE_ACTIVITY_CLUB_1,
    CREATIVE_ACTIVITY_CLUB_2,
    CREATIVE_ACTIVITY_CLUB_3,
    ;

    companion object {
        fun getClubPriorities(): List<ClubPriority> =
            listOf(CREATIVE_ACTIVITY_CLUB_1, CREATIVE_ACTIVITY_CLUB_2, CREATIVE_ACTIVITY_CLUB_3)
    }
}
